### PR TITLE
[11.0] [MIG] sale variant configurator

### PR DESCRIPTION
**Pull request by angelmoya**

### CODE REVIEW

**Comment by angelmoya** [comment]
Depends on #100 

cc @pedrobaeza

**Review by tschanzt — APPROVED** [review]
Looks good. Dates in header are outdated and you could add yourself to the contributors.

File: sale_variant_configurator/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "version": "11.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": [
+        "sale",
+        "product_variant_configurator",
+    ],
+    "author": "OdooMRP team,"
+              "AvanzOSC,"
+              "Tecnativa,"
+              "Odoo Community Association (OCA)",
```
Comment (ernestotejeda) [inline sale_variant_configurator/__manifest__.py:18]: Add a space after each comma that separates the names of the companies. Eg: 'AvanzOSC, '

File: sale_variant_configurator/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "category": "Sales Management",
+    "website": "http://www.odoomrp.com",
```
Comment (ernestotejeda) [inline sale_variant_configurator/__manifest__.py]: https://github.com/OCA/product-variant

**Review by ernestotejeda — CHANGES_REQUESTED** [review]

File: sale_variant_configurator/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply sale_variant_configurator/__manifest__.py:18]: Not needed

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**